app/controllers/users_controller.rb

```diff
@@ -24,6 +24,7 @@ def register
 
   # action to render register form
   def registration
+    @categories = %w[participant organizer maintainer]
```

Comment:
We should make this a hash, so the dropdown can show nicer formatted names
participant => Participant
organizer => Event Organizer
maintainer => Maintainer